lib/fork.js

```diff
@@ -50,14 +50,14 @@ module.exports = function (file, opts) {
 	var relFile = path.relative('.', file);
 
 	var exiting = false;
-	var send = function (ps, name, data) {
```

Comment:
the only place you could make the argument things are "more complex" anywhere is this line, and I don't think it's that bad